packages/builders/scripts/inter-protocol/replace-feeDistributor-combo.js

```diff
@@ -0,0 +1,80 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { getManifestForReplaceFeeDistributor } from '@agoric/inter-protocol/src/proposals/replace-fee-distributor.js';
+import { SECONDS_PER_HOUR } from '@agoric/inter-protocol/src/proposals/econ-behaviors.js';
+
+const configurations = {
```

Comment:
So I'll bite. Why do these config need to be hard coded in the builder? Can't they be restored from whatever params are currently on the respective chain, removing the need for variants?

Reply:
The feeDistributor in `a3p-integration` currently has the same parameter settings as on mainNet. We want to run tests that verify that fees are distributed. Without a  change,  the test would have to wait an hour. We make similar changes with changes to timing for tests for other vats, like auctions.

Reply:
And we can't update these parameters outside of the core proposal, through a core eval or other governance vote, either before the upgrade, or after in the test?

Reply:
FeeDistributor is not a governed contract. (!?!) The collectionInterval can only be modified by restarting/upgrading the contract.

Reply:
Could the contract store the interval in baggage so it can have a value to start from if not in vat params?

Reply:
> Could the contract store the interval in baggage so it can have a value to start from if not in vat params?

It sounds like you're proposing a poor person's fallback to replace governed params. If it's worth making any change here, I'd just make it a governed contract. The contract has no state that we need to preserve.